addons/mail/static/src/core/common/message_in_reply.xml

```diff
@@ -17,9 +17,12 @@
                                     <t t-out="props.message.parent_id.inlineBody"/>
                                     <em t-if="props.message.parent_id.edited" class="smaller fw-bold text-500"> (edited)</em>
                                 </t>
-                                <t t-elif="props.message.parent_id.attachment_ids.length > 0">
-                                    <span class="me-2 fst-italic">Click to see the attachments</span>
-                                    <i class="fa fa-image"/>
+                                <t t-else="">
+                                    <span class="fst-italic">Click to see the attachments</span>
+                                </t>
+                                <t t-if="props.message.parent_id.hasAttachments">
+                                    <span class="ms-2"/>
+                                    <i class="position-absolute top-0 end-0 fa fa-image" />
```

Comment:
Instead of `fa fa-image`, can use `message.parent_id.previewIcon`: this already shows the icon based on message content, i.e. with image, voice message, video, any other file, etc.

Reply:
I didn't know about this, very useful in this case.